Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Getting Started with the MRTK
```

Comment:
Should this PR remove the typo-ed file as well?

Reply:
@hferrone

Reply:
@keveleigh I'm not sure what you mean by remove typo-ed file? GettingStartedWithTheMRTK.md file needs to remain active for anyone who had it bookmarked, so as not to send them to a 404 error page.

Reply:
Right, but as far as I can tell, the typo-ed `GetingStartedWithTheMRTK.md` exists side-by-side with the fixed filename in this PR: 
![image](https://user-images.githubusercontent.com/3580640/94472008-f56cfe00-017e-11eb-8234-5799fa28b55e.png)
We probably only want the fixed one, right?